lighthouse-core/audits/time-to-first-byte.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const Util = require('../report/v2/renderer/util');
+const URL = require('../lib/url-shim');
+
+const TTFB_THRESHOLD = 200;
```

Comment:
Are we intending to run this audit only on unthrottled runs? Won't it flag all resources with the default throttling settings on?

Reply:
I believe I don't understand this comment. TTFB should be below 200, this is merely a constant that I use in the audit.

Reply:
Well if throttling is on won't TTFB always be at least as high as our throttling value?

Reply:
that's a really good question. I believer you are right. Should we higher the max ttfb? I believe @paulirish suggested using 200ms

Reply:
When throttling is the TTFB that we observe should be minimum 150ms. And if there was any server latency, that is just absorbed in there.

So if the observed TTFB is > 151ms, then ALL of that was real network latency + server response.

That much sound correct to yall?

Then, 200 is a number we can evaluate separate from our throttling and we just have to decide when its slow enough for us to care.
PSI uses either 200 or 400, I forget.

Reply:
Ah based on https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/emulation.js#L47  our set latency should be 562ms. So whatever i just say but s/150/562/  :)

well...  then our threshold has to be like 600ms  
eesh.